src/components/03-containers/three-column-container/_three-column-container.scss

```diff
@@ -0,0 +1,26 @@
+.sfgov-container-three-column {
+  flex-direction: column;
+  display: flex;
+  flex-wrap: wrap;
+  @include media($bp-narrow) {
+    flex-direction: row;
+  }
+  > div {
+    width: 100%;
+    margin-bottom: 10px;
+    @include media($bp-narrow) {
+      width: 33.33%;
+      display: flex;
+      align-items: stretch;
+      &:nth-child(3n+3) {
+        > * {
```

Comment:
Would it be possible to convert these selectors into classes? Pretty high specificity here.

Reply:
i think we talked about this in person, but i forgot where we landed.  The reason i did it here in css was so that it wouldn't need to add a `third-child` class in the markup, which would require some calculation

Reply:
To sum up [our Slack convo:](https://sfdigitalservices.slack.com/archives/CEERV5DCG/p1573175275001200?thread_ts=1573170435.000100&cid=CEERV5DCG) it's fine to leave this as is.

Might be a bit more readable if you combined lines 15 and 16, though.